### DISCUSSION TITLE
Github Desktop History

### DISCUSSION THREAD

**Discussion by alastair77** [community/community#24090] [Apps, API and Webhooks]
<p>I’m tryng to find how to see all the changes made to one specific file in Github Desktop. I know there is a history option that shows all the changes in all the files of the project, but that’s not very helpfull.</p>
<p>Thanks a lot</p>

**Comment by steveward** [comment, marked as answer]
<p>Welcome <a class="mention" href="/alastair77">@alastair77</a>. The specific feature you are looking for would be an implementation of <code>git blame</code>, which GitHub Desktop does not currently have. At this time we do not have any plans to implement this feature, but please feel free to add feedback to <a href="https://github.com/desktop/desktop/issues/2310">this issue</a>. We’d appreciate hearing the specific benefit this would provide to you if we added it to GitHub Desktop.</p>
<p>I’d also like to mention that GitHub has a blame feature – see <a href="https://docs.github.com/en/free-pro-team@latest/github/managing-files-in-a-repository/tracking-changes-in-a-file">this help document</a> for more information about that.</p>
